CI-CD/AzureDevOps/azure-pipelines_dotnet_sbom.yml

```diff
@@ -53,5 +53,5 @@ steps:
     WS_APIKEY: $(APIKEY)
     WS_USERKEY: $(USERKEY)
 
-- publish: $(System.DefaultWorkingDirectory)/whitesource
```

Comment:
Keep it "whitesource" as this folder is already created by the UA (and adding the logs)